src/Test.UnitTests.Sarif.Multitool/TestData/QueryCommand/property-bag-queries.sarif

```diff
@@ -62,7 +62,7 @@
           },
           "properties": {
             "comment": "Case-insensitive match.",
-            "name": "terisa"
+            "name": "Calliope"
```

Comment:
>Calliope [](http://example.com/codeflow?start=21&length=8)

question, it was 1 terisa with lower t and the other one was with upper T

Reply:
Yes. Not sure how I missed this one, thanks.